homeassistant/components/ebusd/services.yaml

```diff
@@ -0,0 +1,6 @@
+write:
+  description: Call ebusd write command.
+  fields:
+    call:
```

Comment:
We should document the service data parameters here. See other components' services.yaml files for examples.